File: tests/destination/ts/General.ts
import { Vue } from 'vue';

export class General {
    /** En translation: 30 days en 7118 */
    get _30dayss(): string {
        return Vue.i18n.translate('_30dayss', Vue.i18n.locale());
    }
    /** En translation: 7 days en */
    get _7days(): string {
        return Vue.i18n.translate('_7days', Vue.i18n.locale());
    }
}

File: tests/destination/cs/General.cs
using System.Globalization;
using Agrivi.Translations.Helpers;

namespace Agrivi.Translations.Resources.Modules 
{
	public static class General 
	{
        private const TranslationModule _module = TranslationModule.General;

		public static string GetString(string key, CultureInfo cultureInfo = null) => typeof(General).GetString(key, cultureInfo);

		public static TranslationBase _30dayss = new TranslationBase(_module, "_30dayss");
		public static TranslationBase _7days = new TranslationBase(_module, "_7days");
    }
}